multinet/multinet/db.py

```diff
@@ -1,11 +1,30 @@
 """Low-level database operations to fulfill GraphQL queries."""
 import os
+import sys
 
 from arango import ArangoClient
+from requests.exceptions import ConnectionError
+
+from flask import current_app as app
```

Comment:
If this is only needed for the logging line below, let's get rid of this as well.

Reply:
Removed.